setup.py

```diff
@@ -11,11 +11,7 @@
 
 import os
 import sys
-try:
-    from setuptools import setup, Extension
-except ImportError:
-    from distutils.core import setup, Extension
-
+from setuptools import setup, Extension
```

Comment:
I'm eager to do this only if a brand new Python installation (2.6 included) has setuptools installed.